### ISSUE TITLE
Audio Cogs Unable to Load

### ISSUE THREAD

**Issue by wakarimas** [Cog-Creators/Red-DiscordBot#2066]
# Other bugs

 

#### What were you trying to do?
Trying to migrate from V2 to V3. And then it was unable to load audio cogs. similar to [#2033](https://github.com/Cog-Creators/Red-DiscordBot/issues/2033)

#### What were you expecting to happen?

Expected the audio cogs to load and able to run functionally

#### What actually happened?

Failed to loads. The log shows something like this

```
[29/08/2018 22:09] ERROR core_commands _load 97: Package loading failed
Traceback (most recent call last):
  File "G:\discordbot\wakaisbot5\lib\site-packages\redbot\core\core_commands.py", line 95, in _load
    await bot.load_extension(spec)
  File "G:\discordbot\wakaisbot5\lib\site-packages\redbot\core\bot.py", line 226, in load_extension
    await lib.setup(self)
  File "G:\discordbot\wakaisbot5\lib\site-packages\redbot\cogs\audio\__init__.py", line 53, in setup
    await start_lavalink_server(bot.loop)
  File "G:\discordbot\wakaisbot5\lib\site-packages\redbot\cogs\audio\manager.py", line 89, in start_lavalink_server
    java_available, java_version = await has_java(loop)
  File "G:\discordbot\wakaisbot5\lib\site-packages\redbot\cogs\audio\manager.py", line 49, in has_java
    version = await get_java_version(loop)
  File "G:\discordbot\wakaisbot5\lib\site-packages\redbot\cogs\audio\manager.py", line 62, in get_java_version
    loop=loop,
  File "c:\users\faishal akbar ilham\appdata\local\programs\python\python37\Lib\asyncio\subprocess.py", line 217, in create_subprocess_exec
    stderr=stderr, **kwds)
  File "c:\users\faishal akbar ilham\appdata\local\programs\python\python37\Lib\asyncio\base_events.py", line 1516, in subprocess_exec
    bufsize, **kwargs)
  File "c:\users\faishal akbar ilham\appdata\local\programs\python\python37\Lib\asyncio\base_events.py", line 444, in _make_subprocess_transport
    raise NotImplementedError
NotImplementedError
```

#### How can we reproduce this issue?

Here additional info for what python, redbot, java and pip freeze:

python Version: 3.7.0 running on virtualenv
Redbot Version: Red Bot Version: 3.0.0b20
Java Version: 
> java -showversion
java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)

Pip freeze: 

> aiohttp==3.3.2
aiohttp-json-rpc==0.11
appdirs==1.4.3
async-timeout==3.0.0
attrs==18.1.0
chardet==3.0.4
colorama==0.3.9
discord.py==1.0.0a0
fuzzywuzzy==0.16.0
idna==2.7
idna-ssl==1.1.0
motor==2.0.0
multidict==4.3.1
pymongo==3.7.1
PyYAML==3.13
raven==6.9.0
raven-aiohttp==0.7.0
Red-DiscordBot==3.0.0b20
Red-Lavalink==0.1.2
websockets==6.0
yarl==1.2.6

**Comment by endms3231** [comment]
i had this problem too...

**Comment by aikaterna** [comment]
Please give #2062 a try and report your findings on that issue or wait for that PR to be merged.

**Comment by endms3231** [comment]
Okay

Windows 10용 메일에서 보냄

보낸 사람: aikaterna
보낸 날짜: 2018년 9월 1일 토요일 오후 9:49
받는 사람: Cog-Creators/Red-DiscordBot
참조: 이리에; Comment
제목: Re: [Cog-Creators/Red-DiscordBot] Audio Cogs Unable to Load (#2066)

Please give #2062 a try and report your findings on that issue or wait for that PR to be merged.

**Comment by Tobotimus** [comment]
Resolved by #2062.
